packages/headless/src/test/mock-search-response.ts

```diff
@@ -13,6 +13,13 @@ export function buildMockSearchResponse(
     totalCountFiltered: 0,
     facets: [],
     queryCorrections: [emptyCorrection()],
+    questionAnswer: {
+      answerSnippet: '',
+      documentId: {contentIdKey: '', contentIdValue: ''},
+      question: '',
+      relatedQuestions: [],
+      score: 0,
+    },
```

Comment:
If we created a mock function, we could reuse it here and the file above.

If the api changes, we only need to update one place.
When unit testing, it would be easy to create mock `questionAnswer` data objects.